lib/instrumentation/index.js

```diff
@@ -164,6 +167,13 @@ Instrumentation.prototype.start = function () {
     }
   }
 
+  const lambdaHandlerInfo = getLambdaHandler(process.env)
+  if (lambdaHandlerInfo) {
+    this.addPatch(
+      lambdaHandlerInfo.filePath,
+      require('./modules/_lambda-handler')
+    )
+  }
```

Comment:
Was there a particular reason to do this in `.start()` rather than in the constructor where `addPatch` is called on all the static `MODULES`?

Reply:
The task at hand (wrapping a module dynamically, and wrapping a module that's required in as a full file path outside of the application's node_modules folder) felt like more of a special case that belonged with applying the other special cases from configuration.

Reply:
Okay. I was asking in case I was missing some reason this depended on being run after agent.start(...) had been called.